### PR TITLE
No Bug: Don't generate Info.plist for BraveWidgets target

### PR DESCRIPTION
**Pull request by kylehickinson**
## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: Client.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -306,7 +306,6 @@
 		275965E224EEC4EA0051A827 /* FeedFillStrategyTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 275965E124EEC4EA0051A827 /* FeedFillStrategyTests.swift */; };
 		275C6FBF2731D1A500AB2FC7 /* RpcServiceExtensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 275C6FB32731D1A500AB2FC7 /* RpcServiceExtensions.swift */; };
 		2760056625D1FFF500D47A75 /* AddFeedToBraveNewsActivity.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2760056525D1FFF500D47A75 /* AddFeedToBraveNewsActivity.swift */; };
-		2760D2BF215ACCE20068E131 /* BundleExtensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2760D2BE215ACCE20068E131 /* BundleExtensions.swift */; };
```
Comment (kylehickinson) [inline Client.xcodeproj/project.pbxproj:309]: Not sure what these are, they keep showing up anytime I touch project.pbxproj so I think they are being cleaned up by Xcode and were missed in a previous PR?

Comment (Brandon-T) [reply Client.xcodeproj/project.pbxproj:309]: This is happening to me as well. I assume it's fine to remove tbh. The code still compilers and works fine. At least locally it does.

**Review by Brandon-T — APPROVED** [review]

File: Client.xcodeproj/project.pbxproj (hunk 1 above)
Comment (soner-yuksel) [reply Client.xcodeproj/project.pbxproj:309]: Same

**Review by soner-yuksel — APPROVED** [review]

File: Client.xcodeproj/project.pbxproj (hunk 1 above)
Comment (StephenHeaps) [reply Client.xcodeproj/project.pbxproj:309]: +1

**Pull request merged**